scripts/qa_crowbarsetup.sh

```diff
@@ -280,11 +280,23 @@ function add_mount()
     local nfssrc="$2"
     local targetdir="$3"
     local zypper_alias="$4"
+
     if [ -n "${localreposdir_target}" ]; then
-        [ -n "${bindsrc}" ] && add_bind_mount "${localreposdir_target}/${bindsrc}" "${targetdir}"
+        if [ -z "$bindsrc" ]; then
+            complain 50 "BUG: add_mount() called with empty bindsrc parameter" \
```

Comment:
This would throw errors in those "if ! $longdistance" cases (which could need a review and overhaul anyway), but merging this would break mkcloud atm